data_collection/api/tracking_api.yaml

```diff
@@ -296,92 +338,102 @@ paths:
 
             This is used to identify further actions performed during that page view (e.g. events) as part of
             single page view.
-          type: string
-          format: hexadecimal
-          minimum: 6
-          maximum: 6
+          schema:
+            type: string
+            format: hexadecimal
+            minimum: 6
+            maximum: 6
           example: ff5c6d
         - name: idgoal
           in: query
-          #TODO: Unify Ecommerce naming scheme. Rest of parameters refer to Ecommerce order, not cart.
+          #TODO: Unify E-commerce naming scheme. Rest of parameters refer to E-commerce order, not cart.
           description: |
-            Goal ID. Signifies that a goal has been reached and tracks its conversion.
+            ID of the converted goal. Value `0` indicates that it's an E-commerce event.
```

Comment:
Looks like information about `0` is repeated

Reply:
Yes, I will restore the old description.